assets/dev/js/frontend/handlers/hotspot.js

```diff
@@ -0,0 +1,80 @@
+export default class Hotspot extends elementorModules.frontend.handlers.Base {
+    getDefaultSettings() {
+        return {
+            selectors: {
```

Comment:
1. Use `e-` prefix instead of `elementor-`
2. Use the prefix anywhere
3. Use the BEM code standard for class names
4. Don't use the `var` keyword
5. Use only arrow functions
6. Use `String.prototype.startWith` instead of `String.prototype.indexOf`

Reply:
Done